sdk/appcenter/src/test/java/com/microsoft/appcenter/channel/DefaultChannelTest.java

```diff
@@ -1190,4 +1195,55 @@ public void setNetworkRequest() {
         channel.setNetworkRequests(true);
         verify(mockIngestion, times(2)).sendAsync(anyString(), any(UUID.class), any(LogContainer.class), any(ServiceCallback.class));
     }
+
+    @Test
+    public void testConcurrentIterableDataStructureUpdatesSuccess()
```

Comment:
These tests don't test our code. Let's test the modified property `mGroupStates` instead of an abstract HashMap and ConcurrentHashMap objects.

Reply:
done